modules/ROOT/pages/api-gateway-encryption-mule4.adoc

```diff
@@ -4,10 +4,9 @@ include::_attributes.adoc[]
 endif::[]
 
 
-Gateway startup encryption enables you to encrypt sensitive information stored by Mule runtime engine, such as policies, contracts, and the credentials required to configure the connection between your Mule application and Anypoint Platform.
-
-This feature is supported in Mule 4.2.0 and later.
+Gateway startup encryption enables you to encrypt sensitive information stored by Mule runtime engine, such as policies, contracts, and the credentials required to configure the connection between your Mule runtime engine (Mule) application and Anypoint Platform.
 
+This feature is supported in only Mule 4.2.0 and later. If a policy supporting encryption, such as Basic Authentication 1.2.0 is applied to an API that is running in Mule versions earlier than 4.2.0, the policy is not encrypted.
```

Comment:
```suggestion
This feature is supported only in Mule 4.2.0 and later. If a policy supporting encryption, such as Basic Authentication 1.2.0 is applied into an API that is running in Mule versions earlier than 4.2.0, the policy will not be encrypted.
```

About the example of Basic Auth, I don't know what is better: to give a particular example, or to give a reference to the `Policies That Support Encryption` section. I'll leave it up to you, but I had this new idea. 😃